app/manifest/v3/_base.json

```diff
@@ -47,7 +47,16 @@
   ],
   "default_locale": "en",
   "description": "__MSG_appDescription__",
-  "host_permissions": ["file://*/*", "http://*/*", "https://*/*"],
+  "host_permissions": [
+    "http://localhost:8545/",
+    "file://*/*",
+    "http://*/*",
+    "https://*/*",
+    "*://*.eth/",
```

Comment:
I don't think we need this or the 3 URLs following after we are using 
```
    "http://*/*",
    "https://*/*",
```

keeping these specific URLs to be more explicit

Reply:
This can be complete list
```
    "file://*/*",
    "http://*/*",
    "https://*/*",
    "*://*.eth/",
```

Reply:
ok 👍  https://github.com/MetaMask/metamask-extension/pull/15777/commits/4721af8c9de169c0f208d5d5d4b1babb7f2e06d0

Reply:
We'll need to restore those URLs once we drop the wildcard host permissions that we needed just for dynamic contentscript injection. We won't need that once the Chromium team fixes MAIN-world contentscript support.

But dropping them makes sense for now